pipeline/build/ldflags.go

```diff
@@ -20,6 +20,9 @@ func ldflags(ctx *context.Context) (string, error) {
 		Version: ctx.Git.CurrentTag,
 		Date:    time.Now().UTC().Format(time.RFC3339),
 	}
+	if ctx.Snapshot {
```

Comment:
i think here the `data` is actually wrong.

We I'll push a fix to that, then you will be able to just remove this `if` stm. :)